compiler/parser/parser.peg

```diff
@@ -88,14 +88,16 @@ Operation
       RETURN(MAP("kind": "From", "trunks": trunks))
     }
   / op:Operator
-  / f:Function &EndOfOp { RETURN(f) }
   / a:OpAssignment &EndOfOp { RETURN(a) }
-  / a:Aggregation &EndOfOp  { RETURN(a) }
-  / r:Record &EndOfOp {
-      RETURN(MAP("kind": "Yield", "exprs": ARRAY(r)))
+  / !(Function EndOfOp) a:Aggregation &EndOfOp  { RETURN(a) }
+  / "search" _ expr:SearchBoolean {
+      RETURN(MAP("kind": "Where", "expr": expr))
     }
-  / expr:SearchBoolean !AggGuard {
-      RETURN(MAP("kind": "Filter", "expr": expr))
+  / expr:SearchBoolean {
+      RETURN(MAP("kind": "OpExpr", "expr": expr))
+    }
+  / expr:Expr {
+      RETURN(MAP("kind": "OpExpr", "expr": expr))
```

Comment:
Maybe combine?
```suggestion
  / expr:(Expr / SearchBoolean) {
      RETURN(MAP("kind": "OpExpr", "expr": expr))
```

Reply:
I think I will leave it since the PEG order matters and separate lines stress that.  You introduced a bug where identifiers will be matched before keywords so I believe `foo` would be `yield foo` instead of `search("foo")`